From 5ace470d4e5c7013dd0bc3b162d6d2188129f352 Mon Sep 17 00:00:00 2001
From: Pedro Falcato <pedro.falcato@gmail.com>
Date: Sat, 22 Mar 2025 20:18:03 +0000
Subject: [PATCH] Remove some deps out

A bunch of them....

Signed-off-by: Pedro Falcato <pedro.falcato@gmail.com>
---
 librepo.spec | 42 +++++++++++++++++++++---------------------
 1 file changed, 21 insertions(+), 21 deletions(-)

diff --git a/librepo.spec b/librepo.spec
index cdde775..b19dd72 100644
--- a/librepo.spec
+++ b/librepo.spec
@@ -41,25 +41,25 @@ Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  cmake
 BuildRequires:  gcc
 BuildRequires:  check-devel
-BuildRequires:  doxygen
-BuildRequires:  pkgconfig(glib-2.0) >= 2.66
+#BuildRequires:  doxygen
+#BuildRequires:  pkgconfig(glib-2.0) >= 2.66
 %if %{with use_gpgme}
-BuildRequires:  gpgme-devel
+#BuildRequires:  gpgme-devel
 %else
-BuildRequires:  pkgconfig(rpm) >= 4.18.0
+#BuildRequires:  pkgconfig(rpm) >= 4.18.0
 %endif
-BuildRequires:  libattr-devel
+#BuildRequires:  libattr-devel
 BuildRequires:  libcurl-devel >= %{libcurl_version}
-BuildRequires:  pkgconfig(libxml-2.0)
-BuildRequires:  pkgconfig(libcrypto)
+#BuildRequires:  pkgconfig(libxml-2.0)
+#BuildRequires:  pkgconfig(libcrypto)
 %if %{need_selinux}
-BuildRequires:  pkgconfig(libselinux)
+#BuildRequires:  pkgconfig(libselinux)
 %endif
-BuildRequires:  pkgconfig(openssl)
+#BuildRequires:  pkgconfig(openssl)
 %if %{with zchunk}
-BuildRequires:  pkgconfig(zck) >= 0.9.11
+#BuildRequires:  pkgconfig(zck) >= 0.9.11
 %endif
-Requires:       libcurl%{?_isa} >= %{libcurl_version}
+#Requires:       libcurl%{?_isa} >= %{libcurl_version}
 
 %if %{with sanitizers}
 BuildRequires:  libasan
@@ -85,10 +85,10 @@ Development files for librepo.
 Summary:        Python 3 bindings for the librepo library
 %{?python_provide:%python_provide python3-%{name}}
 BuildRequires:  python3-devel
-BuildRequires:  python3-gpg
-BuildRequires:  python3-pyxattr
-BuildRequires:  python3-requests
-BuildRequires:  python3-sphinx
+#BuildRequires:  python3-gpg
+#BuildRequires:  python3-pyxattr
+#BuildRequires:  python3-requests
+#BuildRequires:  python3-sphinx
 Requires:       %{name}%{?_isa} = %{version}-%{release}
 # Obsoletes Fedora 27 package
 Obsoletes:      platform-python-%{name} < %{version}-%{release}
@@ -101,23 +101,21 @@ Python 3 bindings for the librepo library.
 %autosetup -p1
 
 %build
-%cmake \
+%cmake . -DCMAKE_INSTALL_PREFIX=/usr \
     -DWITH_ZCHUNK=%{?with_zchunk:ON}%{!?with_zchunk:OFF} \
     -DUSE_GPGME=%{?with_use_gpgme:ON}%{!?with_use_gpgme:OFF} \
     -DUSE_RUN_GNUPG_USER_SOCKET=%{?with_run_gnupg_user_socket:ON}%{!?with_run_gnupg_user_socket:OFF} \
     -DWITH_SANITIZERS=%{?with_sanitizers:ON}%{!?with_sanitizers:OFF} \
+    -DENABLE_DOCS=FALSE -DENABLE_TESTS=FALSE -DENABLE_EXAMPLES=OFF \
 %if %{need_selinux}
     -DENABLE_SELINUX=ON
 %else
     -DENABLE_SELINUX=OFF
 %endif
-%cmake_build
-
-%check
-%ctest
+%make_build
 
 %install
-%cmake_install
+%make_install
 
 %if 0%{?rhel} && 0%{?rhel} <= 7
 %post -p /sbin/ldconfig
@@ -136,6 +134,8 @@ Python 3 bindings for the librepo library.
 %{_libdir}/pkgconfig/%{name}.pc
 %{_includedir}/%{name}/
 
+%define python3_sitearch /usr/lib/python3.10/site-packages
+
 %files -n python3-%{name}
 %{python3_sitearch}/%{name}/
 

base-commit: 373c77e0f7dabc60b5147a046a54d52071b58fa2
-- 
2.46.2

